OrbitGl/SchedulerTrack.cpp

```diff
@@ -0,0 +1,13 @@
+#include "SchedulerTrack.h"
+#include "TimeGraph.h"
+
+void SchedulerTrack::Draw(GlCanvas* /*a_Canvas*/, bool /*a_Picking*/) {
```

Comment:
Nit: Above you use UNUSED, here you comment out names. Should be consistent.

Reply:
Done above, removed UNUSED.